tests/js/spec/components/events/interfaces/utils.spec.jsx

```diff
@@ -60,4 +63,33 @@ describe('components/interfaces/utils', function() {
       );
     });
   });
+
+  describe('objectToSortedTupleArray()', function() {
+    it('should convert a key/value object to a sorted array of key/value tuples', function() {
+      // expect(
```

Comment:
probably drop this commented code or fix it, it was commented out
in this pr, probably for sake of getting this out of the way, but no reason to kick it down the road.
https://github.com/getsentry/sentry/pull/5252/files#diff-81396dee1890169987cb2d097545b261L26

Reply:
I'll fix this in a separate PR.